src/modules/RA/DataLayer/components/StyleField.js

```diff
@@ -9,9 +9,6 @@ const validate = value => {
   if (!value.type) {
     return 'type not found';
   }
-  if (!value.paint) {
-    return 'paint not found';
-  }
```

Comment:
Sure of this?

Reply:
Also, maybe some translations would be appropriate.

Reply:
Yes sure, we have case in PACA where only icon are added and paint is empty. But you are right for translation.

Reply:
done